client/state/form/packages/reducer.js

```diff
@@ -10,6 +10,10 @@ import {
 import omitBy from 'lodash/omitBy';
 import isNull from 'lodash/isNull';
 
+const isNullOrEmpty = ( value ) => {
```

Comment:
I think lodash's [`isEmpty`](https://lodash.com/docs#isEmpty`) would be better fit here